examples/calendar/backend/services/time/controller.ts

```diff
@@ -0,0 +1,92 @@
+import { NextFunction, Request, Response } from "express";
+//import { User } from "../../models/user.model";
+import { Time } from "../../models/time.model";
+import { Op } from "sequelize";
+
+// Create time availablity for the user
+export const create = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  if ((req as any).user.payload.id !== +req.body.userId) {
```

Comment:
what if user or payload is undefined?

Reply:
hm not sure but i think it would fail at jwt middleware level and never reach this one if that was the case

Reply:
@skosito yes, thus we can directly use 
`const userId = req.user.payload.id` as requested in other review comments. I have modified that pushed new changes.